### DISCUSSION TITLE
Configuration for temperature setpoint

### DISCUSSION THREAD

**Discussion by Comiter** [BeamCtrl/Airiana#25] [Q&A]
Hi,
just installed the Airana on an old Rpi, connected to my SAVE VTR 500 through a IAM configured as a Modbus gateway.
So far so good, and the Airana is up and running. However, the setpoint of 24 degrees are a bit high, and there are no setting for this in the configuration as I can see. Also, it seems like the Airana is activating the electric heater in the ventilation unit to keep the air temperature at 24 degrees.
I ideally would like to be able to change these on the fly, to be able to let it follow my temperatture setting of my underfloor heating settings (minus one degree). 
I guess it would not be as easy to just change the self.target from the default values of 24/23/20.7 degrees, as doing so Airiana fail to start.

**Comment by BeamCtrl** [comment]
Hello, TLDR; YES, you just need to change the self.target to change the behavior, the changes will however be over written if you have auto update enabled in the crontab file, use `crontab -e` to edit. 
The electric heater should self disable on startup if you have a non Savecair unit, if not you will need to disable it yourself it should never enable it.

I have a plan to implement a manual target temp update function from the WebHMI in the future aswell as electric heat disabling on savecair units.

hope this answers you question.
//Daniel

**Reply by Comiter** [reply]
Hi Daniel,
thank you for the answer! THen I'll disable the auto update and give it a try.
